src/couch/src/couch_httpd_auth.erl

```diff
@@ -186,6 +188,23 @@ proxy_auth_user(Req) ->
             end
     end.
 
+jwt_authentication_handler(Req) ->
+    RequiredClaims = ["sub", "exp"],
+    case { config:get("chttpd", "jwt_secret"), header_value(Req, "Authorization") } of
```

Comment:
our code style does not have spaces around braces, so `case {config:get...` etc.